src/App.js

```diff
@@ -1,129 +1,53 @@
-
-    const addTodo = async (title) => {
-        const newTodo = {
-            fields: {
-                title: title, // Airtable field name
-            }
-        };
-
-        const options = {
-            method: 'POST',
-            headers: {
-                'Authorization': `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`,
-                'Content-Type': 'application/json'
-            },
-            body: JSON.stringify(newTodo)
-        };
-        try {
-            const response = await fetch(url, options);
-            if (!response.ok) {
-                throw new Error(`HTTP error! status: ${response.status}`);
-            }
 
-            const data = await response.json();
-            setTodoList([...todoList, {
-                title: data.fields.Title,
-                id: data.id,
-                completed: data.fields.completed || false,
-            }]);
-
-        } catch (error) {
-            console.error('Error adding todo:', error);
-        }
-    };
-
-    const removeTodo = (id) => {
-        const updatedList = todoList.filter(todo => todo.id !== id);
-        setTodoList(updatedList);
-    };
-
-    const toggleTodoCompletion = (id) => {
-        const updatedTodoList = todoList.map((todo) =>
-            todo.id === id ? {...todo, completed: !todo.completed} : todo
-        );
-
-        const sortedTodoList = updatedTodoList.sort((a, b) =>
-            a.completed === b.completed ? 0 : a.completed ? 1 : -1
-        );
-
-        setTodoList(sortedTodoList);
-    };
-
-    const reorderTodo = (newTodoList) => {
-        setTodoList(newTodoList);
-    };
+const TableChooser = () => {
+    const [tableName, setTableName] = useState(process.env.REACT_APP_TABLE_NAME);
+    return (
+        <div>
+            <select
+                id="selectField"
+                value={tableName}
+                onChange={(event) => setTableName(event.target.value)}
+            >
+                <option value={process.env.REACT_APP_TABLE_NAME}>Todo List</option>
+                <option value="List2">Todo List 2</option>
+            </select>
+            <TodoContainer tableName={tableName}/>
+        </div>
+    );
+};
 
+function App() {
     return (
         <BrowserRouter>
-          <Routes>
-            <Route path="/" element={
-              <div className={styled.container}>
-                <h1>ToDo List</h1>
-                <AddTodoForm onAddTodo={addTodo}/>
-                {isLoading ? (
-                  <p>Loading ...</p>
-                ) : (
-                  <TodoList
-                    todoList={todoList}
-                    onRemoveTodo={removeTodo}
-                    toggleTodoCompletion={toggleTodoCompletion}
-                    onReorderTodo={reorderTodo}
-                  />
-                )}
-              </div>
-            } />
-            <Route path="/new" element={<h1>New Todo List</h1>} />
-          </Routes>
+            <TodoCounterProvider>
```

Comment:
This should be added either here OR in index.js, it doesn't need to be both places.